engine/src/main/resources/application.conf

```diff
@@ -36,6 +36,9 @@ system {
 
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
   abort-jobs-on-terminate = false
+
+  // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  maxRetries = 3
```

Comment:
I was thinking the property should be on the backend, but then the "feature as a whole" question gets interesting.  If the engine is made responsible for enforcing this limit, how does it know what the backend's retry limit is?